OpenChart/src/UI/NoteField/NoteFieldView.cs

```diff
@@ -0,0 +1,38 @@
+using Gtk;
+
+namespace OpenChart.UI.NoteField
+{
+    /// <summary>
+    /// A wrapper for the note field which adds extra functionality like scrolling.
+    /// </summary>
+    public class NoteFieldView : IWidget
+    {
+        /// <summary>
+        /// The note field widget.
+        /// </summary>
+        public NoteField NoteField { get; private set; }
+
+        public Scrollable Container { get; private set; }
+        public Widget GetWidget() => Container.GetWidget();
```

Comment:
same here. also I notice in this case you chose to import Gtk, remove for consistency?